src/message/__tests__/messageActionSheet-test.js

```diff
@@ -1,5 +1,6 @@
 // @flow strict-local
 import deepFreeze from 'deep-freeze';
+
```

Comment:
Oh! I didn't know about this!

I have one question - In several files, there is a newline after `// @flow strict-local`, in some files there isn't. I couldn't decipher any pattern. What is the correct practice?

Reply:
> In several files, there is a newline after `// @flow strict-local`, in some files there isn't. I couldn't decipher any pattern. What is the correct practice?

Reasonable question!

I think we don't really have a set style. :-) I can't think of a particular preference I really have, either.

I did just now go look for examples of how I've done it myself, when adding a new file or adding type-checking to a file. I used the command `git lsp --author greg -S '@flow strict-local'` (i.e. `git log --stat -p --author greg -S '@flow strict-local'` but I [have an alias](https://gist.github.com/gnprice/5f71b63464a76bd7e0480f207b1bbff3) for `log --stat -p`), and used ["the secret to `git log -p`"](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/git.md#2-the-secret-to-using-git-log--p) to conveniently browse the results.

The result is... I do both. :smile: